indexprovider/wrapper.go

```diff
@@ -262,6 +263,14 @@ func (w *Wrapper) IndexerAnnounceAllDeals(ctx context.Context) error {
 	return merr
 }
 
+func (w *Wrapper) IndexerAnnounceLatest(ctx context.Context) (cid.Cid, error) {
+	e, ok := w.prov.(*engine.Engine)
+	if !ok {
+		return cid.Undef, fmt.Errorf("index provider is disabled")
+	}
+	return e.PublishLatest(ctx)
```

Comment:
There is a separate function for direct announce over HTTP. Should we check if direct announce is enabled and call that as well as this?

Reply:
Oh interesting.. I guess that could be a separate cli command so that we can debug both explicitly

Reply:
or `--direct` flag?

Reply:
I think I will add a separate command because you also have the option to explicitly specify the url